07-bootstrap-validation.md

```diff
@@ -10,7 +10,7 @@ GitOps allows a team to author Kubernetes manifest files, persist them in their
 - Cluster-wide configuration of Azure Monitor for Containers
 - The workload's namespace named `a0008`
 
-1. Install `kubectl` 1.27 or newer. (`kubectl` supports ±1 Kubernetes version.)
+1. Install `kubectl` 1.28 or newer. (`kubectl` supports ±1 Kubernetes version.)
```

Comment:
are we maintaining the client version n-1 for a reason? Any desire to move over 1.29 instead?

Reply:
That was the guidance I received from @ckittel on a previous PR. 

Also, it appears to be [official policy from Kubernetes](https://kubernetes.io/releases/version-skew-policy/#kubectl)

Reply:
Yeah, while Im aware of how it gets supported, I was wondering why we are prescribing that. Just wanted to learn from this.